### ISSUE TITLE
[Bug] Extra /docs is introduced in the home icon of the breadcrumb during configuration of NgDoc route prefix

### ISSUE THREAD

**Issue by akhileshThapliyal** [ng-doc/ng-doc#167]
### Is this a regression?

No

### Description

Extra /docs URI is suffixed to the home icon href attribute value.

**Current Behavior**: Extra /docs URI is suffixed to the home icon href attribute value. I am trying to leverage the configuration to be able to configure the global route for documentation as per the guidelines here https://ng-doc.com/docs/getting-started/configuration#configure-ngdoc-route-prefix

**Expected Behavior**: It shouldn't add an extra /docs to the route link, this is also apparent in the existing ng-doc website

<img width="1396" alt="image" src="https://github.com/ng-doc/ng-doc/assets/7280745/6f8f1291-ddc3-4afb-bb56-90e3a12b3255">

### Please provide the exception or error you saw

```true
No errors were observed, but logically it shouldn't have the extra /docs.
```

### OS

Unix (Linux, macOS, etc.)

### Browser

Chrome

### Node version

20.9.0

### Anything else?

Ng Doc Version: 17.6.10

**Comment by skoropadas** [comment]
Good catch, I'll check it 👍

**Comment by akhileshThapliyal** [comment]
> 🎉 This issue has been resolved in version 17.6.11 🎉
> 
> The release is available on:
> 
> * `v17.6.11`
> * [GitHub release](https://github.com/ng-doc/ng-doc/releases/tag/v17.6.11)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀

Thank you so much, I will try this out. Is it possible to configure the home icon link, I want to be able to direct user to the /root of the domain name and not part of the documentation. For instance, if I am on "https://ng-doc.com/docs/getting-started/installation" can it take me to the "landing" page ie., "https://ng-doc.com" instead of "https://ng-doc.com/docs"? Is there a way to configure this?

The reason is if we have a landing page the routes are "/" or "" and "/docs" for the documentation. Hence, for a user home should mean the landing page and not the base page of the docs. Let me know if this is something which could be achieved.

**Comment by akhileshThapliyal** [comment]
> > 🎉 This issue has been resolved in version 17.6.11 🎉
> > The release is available on:
> > 
> > * `v17.6.11`
> > * [GitHub release](https://github.com/ng-doc/ng-doc/releases/tag/v17.6.11)
> > 
> > Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀
> 
> Thank you @skoropadas so much, I will try this out. Is it possible to configure the home icon link, I want to be able to direct user to the /root of the domain name and not part of the documentation. For instance, if I am on "https://ng-doc.com/docs/getting-started/installation" can it take me to the "landing" page ie., "https://ng-doc.com" instead of "https://ng-doc.com/docs"? Is there a way to configure this?
> 
> The reason is if we have a landing page the routes are "/" or "" and "/docs" for the documentation. Hence, for a user home should mean the landing page and not the base page of the docs. Let me know if this is something which could be achieved.

It is working as expected, it is now correctly configuring the home icon to /docs. Thank you.
